src/_03_IntroToStacks/_03_TestMatchingBrackets.java

```diff
@@ -18,10 +18,22 @@ public void testMatchingBrackets() {
 		assertFalse(doBracketsMatch("}{"));
 	}
 
-	// USE A STACK TO COMPLETE THE METHOD FOR CHECKING IF EVERY OPENING BRACKET HAS A MATCHING CLOSING BRACKET
 	private boolean doBracketsMatch(String b) {
-		
-		return false;
+		Stack<String> bracketList = new Stack<>();
+
+		for (int i = 0; i < b.length(); i++) {
+			char c = b.charAt(i);
+			if (c == '{') {

```

Comment:
Can you figure out a way to push `c` directly onto the stack instead of pushing a different string?